server/src/models/room.py

```diff
@@ -0,0 +1,21 @@
+from schematics.models import Model
+from schematics.types import StringType, BooleanType
+from schematics.exceptions import ValidationError
+
+
+class Room(Model):
+    room_name = StringType(required=True)
+    is_private = BooleanType(default=False, required=True)
+    password = StringType(default='')
+
+    def validate_room_name(self, data, value):
```

Comment:
Not necessary method, StringType has `max_length` and `min_length` arguments that you can pass in it
https://github.com/schematics/schematics/blob/master/schematics/types/base.py#L400